<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>501</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>890</width>
    <height>501</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>890</width>
    <height>501</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Student Manager</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(218, 114, 45);
background-color: rgb(255, 170, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="horizontalFrame">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>811</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(220, 115, 45);
background-color: rgb(255, 170, 0);</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="courseViewButton">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);
background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Course View</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Search and Edit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="textOutput">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>71</y>
      <width>821</width>
      <height>391</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ID Number</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Course</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Year</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Sex</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Enrollment Status</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
